CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Model/Coin.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Coin.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-11.
+//
+
+import Foundation
+
+struct Coin: Identifiable, Decodable {
```

Comment:
It is better to make such models immutable, so you define

```
let id: Int
```

Reply:
updated